r/elasticache_replication_group: add tags to state (#12974)
terraform-providers/terraform-provider-aws: website/docs/r/elasticache_replication_group.html.markdown

before being deleted. If the value of SnapshotRetentionLimit is set to zero (0), backups are turned off.
Please note that setting a `snapshot_retention_limit` is not supported on cache.t1.micro or cache.t2.* cache nodes
* `apply_immediately` - (Optional) Specifies whether any modifications are applied immediately, or during the next maintenance window. Default is `false`.

> * `tags` - (Optional) A map of tags to assign to the resource. adding tags to this resource actually adds tags to clusters in the replication group and not to the group itself.

Reviewer: ```suggestion
* `tags` - (Optional) A map of tags to assign to the resource. Adding tags to this resource will add or overwrite any existing tags on the clusters in the replication group and not to the group itself.
```
